README.md

```diff
@@ -1,205 +1,60 @@
-
-    Provides code coverage metrics for Java code via integration with JaCoCo.
-    You can create the report by running the following command:
-
-    ```bash
-      ./gradlew jacocoTestReport
+    ```markdown
+    git clone https://github.com/hmcts/adoption-backend.git
     ```
 
-    The report will be created in build/reports subdirectory in your project directory.
-
-  * io.spring.dependency-management
-
-    https://github.com/spring-gradle-plugins/dependency-management-plugin
-
-    Provides Maven-like dependency management. Allows you to declare dependency management
-    using `dependency 'groupId:artifactId:version'`
-    or `dependency group:'group', name:'name', version:version'`.
-
-  * org.springframework.boot
-
-    http://projects.spring.io/spring-boot/
-
-    Reduces the amount of work needed to create a Spring application
-
-  * org.owasp.dependencycheck
-
-    https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/index.html
-
-    Provides monitoring of the project's dependent libraries and creating a report
-    of known vulnerable components that are included in the build. To run it
-    execute `gradle dependencyCheck` command.
-
-  * com.github.ben-manes.versions
-
-    https://github.com/ben-manes/gradle-versions-plugin
-
-    Provides a task to determine which dependencies have updates. Usage:
-
-    ```bash
-      ./gradlew dependencyUpdates -Drevision=release
-    ```
-
-## Setup
-
-Located in `./bin/init.sh`. Simply run and follow the explanation how to execute it.
-
-## Notes
-
-Since Spring Boot 2.1 bean overriding is disabled. If you want to enable it you will need to set `spring.main.allow-bean-definition-overriding` to `true`.
-
-JUnit 5 is now enabled by default in the project. Please refrain from using JUnit4 and use the next generation
-
-## Building and deploying the application
-
-### Building the application
-
-The project uses [Gradle](https://gradle.org) as a build tool. It already contains
-`./gradlew` wrapper script, so there's no need to install gradle.
+### Using intelliJ IDE
 
-To build the project execute the following command:
+* Open Project with intelliJ IDE
+* Setup ``Run Configuration`` for project. (IntelliJ _should_ automatically detect the `Application.java` file)
+    * For ``Environment variables`` use `spring.profiles.active=local`
+* Click on ``Run`` and the application will be started.
+* Verify application is running by visiting ``http://localhost:4550/health`` from a browser or by using ``curl`` using the CLI
 
-```bash
-  ./gradlew build
-```
-
-### Running the application
+### Using Docker
 
-Create the image of the application by executing the following command:
+#### Building and Running the application
 
 ```bash
-  ./gradlew assemble
+  ./bin/run-in-docker.sh
 ```
 
-Create docker image:
+The above command assumes the following are set as variables `DB_PASSWORD=${DB_PASSWORD}`, `S2S_URL=${S2S_URL}`, `S2S_SECRET=${S2S_SECRET}`
```

Comment:
I still think we can remove - it's all commented out anyway

Reply:
updated